build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.12.9-1
+v1.13.0-1
```

Comment:
are you planning to push this image?

Reply:
was waiting for the upstream scale issue to be resolved, and 1.13.1 to be released before messing with pushing images

Reply:
ACK

Reply:
1.13.1 is out.